detox/android/detox/src/main/java/com/wix/detox/ReactNativeSupport.java

```diff
@@ -0,0 +1,339 @@
+package com.wix.detox;
+
+import android.os.Handler;
+import android.os.Looper;
+import android.support.annotation.NonNull;
+import android.support.test.InstrumentationRegistry;
+import android.support.test.espresso.Espresso;
+import android.support.test.espresso.IdlingResource;
+import android.util.Log;
+
+import com.wix.detox.espresso.ReactBridgeIdlingResource;
+import com.wix.detox.espresso.ReactNativeTimersIdlingResource;
+
+import org.joor.Reflect;
```

Comment:
We may be able to use the same reflection library as we use for our internal reflection mechanism (a small wrapper on top of Apache ReflectionUtils)